components/application-service/overlays/kcp-unstable/apiexport.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: application-service
```

Comment:
btw, the name of the APIExport is "has" and not "application-service". We should align with these names because right now the CPS contains two APIExports for HAS.